taskcluster/translations_taskgraph/parameters.py

```diff
@@ -148,6 +142,23 @@ def get_defaults(_):
                         str: float,
                     },
                 },
+                Optional("pretrained-models"): {
+                    Optional("teacher-base"): {
+                        Required("urls"): [str],
+                        Required("mode"): str,
+                        Required("type"): str,
+                    },
+                    Optional("teacher-finetuned"): {
+                        Required("urls"): [str],
+                        Required("mode"): str,
+                        Required("type"): str,
+                    },
+                    Optional("backwards"): {
```

Comment:
This raises a question: are we going to support testing pretrained model support in the quick CI runs? Currently we don't, as these are absent from the defaults above. (If we are, that could totally be added in a follow-up, or filed as an issue for the future.)

Reply:
Is it possible to do that, but run the usual training if the training task was modified? I could trigger an action on the  push decision.

Reply:
I like the idea...

Reply:
I imagine it would be possible to dynamically choose whether or not to use a pretrained X.

The downside to doing it with an action task is that it can only be done by those without specific permissions. Not the end of the world, I guess, as anyone who could review/merge presumably has those permissions...